modules/nw-sriov-hwol-using-vf-management-port.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="using-vf-improve-network-traffic-performance_{context}"]
+= Using a virtual function to improve network traffic performance
```

Comment:
Might be better as "Improving network traffic performance using a virtual function". Your call.

Reply:
+1

Reply:
Great suggestion, thank you!